test/clj/villagebook/factory.clj

```diff
@@ -18,6 +20,14 @@
   {:name  (fseq :orgname)
    :color (fseq :orgcolor)})
 
+(deffactory :field
+  {:name (fseq :field-name)})
```

Comment:
Shouldn't this also have a `type`?

Reply:
`type` is optional at the moment, it defaults to `text`. Will update this when the type feature is added.